tests/operators/test_agnostic_append.py

```diff
@@ -0,0 +1,116 @@
+import pathlib
+
+import pandas as pd
+from airflow.exceptions import BackfillUnfinished
+
+from astro import sql as aql
+from astro.dataframe import dataframe as adf
+from tests.operators import utils as test_utils
+
+CWD = pathlib.Path(__file__).parent
+import pytest
+
+
+@adf
+def validate_basic(df: pd.DataFrame):
+    assert len(df) == 6
+    assert not df["sell"].hasnans
+    assert df["rooms"].hasnans
+
+
+@adf
+def validate_append_all(df: pd.DataFrame):
+    assert len(df) == 6
+    assert not df["sell"].hasnans
+    assert not df["rooms"].hasnans
+
+
+@adf
+def validate_caste_only(df: pd.DataFrame):
+    assert len(df) == 6
+    assert not df["age"].hasnans
+    assert df["sell"].hasnans
+
+
+@pytest.fixture
+def append_params(request):
+    mode = request.param
+    if mode == "basic":
+        return {
+            "columns": ["sell", "living"],
+        }, validate_basic
+    if mode == "all_fields":
+        return {}, validate_append_all
+    if mode == "with_caste":
```

Comment:
If this test runs against all the databases, we probably do not need to run "basic" and "cast_only" against all the other databases.